src/sdkBase.ts

```diff
@@ -72,6 +74,14 @@ export abstract class SdkBase {
     this.initInternal(opts);
   }
 
+  public isInitialized() {
+    return this._initComplete;
+  }
+
+  public setInitComplete(isInitComplete: boolean) {
```

Comment:
I don't think this should be a public method. Just setting the private variable is fine.